linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,39 @@
+<<<<<<< HEAD
```

Comment:
you need to fix file conflict

e.g. `<<<<<<< HEAD` this is generated by git due to a conflict